src/shared/questions/links.js

```diff
@@ -102,18 +130,30 @@ async function updatelinks(links) {
         message: "Do you want to update another link?",
       },
     ]);
+    
+    if(answers.color !== "") {
+      links[links.length - 1].color = answers.color;
+    }
+
+    if(answers.group !== "") {
+      links[links.length - 1].group = answers.group;
+    }
+
```

Comment:
Hey,
There is no object called `answers` why are checking here?

Reply:
Ok, removed, but I don't understand where do I have to put the if statements